src/Scripting/CSharpTest/ScriptTests.cs

```diff
@@ -944,7 +944,8 @@ public override Stream OpenRead(string resolvedPath)
             catch (Exception ex)
             {
                 // line information is only available when PDBs have been emitted
-                var stackTrace = new StackTrace(ex, needFileInfo: true);
+                var needFileInfo = true;
```

Comment:
Maybe /* needFileInfo: */ is a bit clearer. I would not have guessed this at all.